bin/load_performance.py

```diff
@@ -70,73 +72,80 @@ def fetch_column_field_data(db_path):
     df_column_field = pd.read_sql_query(query, conn)
     return df_column_field
 
+
 def create_performance_tables(merged_data, cf_merged_data, performance_db_path):
     conn = sqlite3.connect(performance_db_path)
-    column_field_table_name = "column_field_summary"
-    cf_merged_data_filtered = cf_merged_data[cf_merged_data['endpoint'].notna()]
-    cf_merged_data_filtered.to_sql(column_field_table_name, conn, if_exists="replace", index=False)
+    column_field_table_name = "endpoint__dataset_resource_summary"
```

Comment:
did you mean to type two underscores trailing `endpoint` here? it's not there for the issue table name so i'm guessing not.